pydevd_plugins/extensions/README.md

```diff
@@ -0,0 +1,30 @@
+Extensions allow extending the debugger without modifying the debugger code. This is implemented with explicit namespace
+packages.
+
+To implement your own extension:
+
+1. Ensure that the root folder of your extension is in sys.path (add it to PYTHONPATH) 
+2. Ensure that your module follows the directory structure below
+3. The ``__init__.py`` files inside the pydevd_plugin and extension folder must contain the preamble below,
+and nothing else.
+Preamble: 
+```python
+try:
+    __import__('pkg_resources').declare_namespace(__name__)
```

Comment:
Please see my previous comment about pkg_resources.   The pkgutil idiom is indeed an error. Thanks for catching it.